src/Native/LibTorchSharp/crc32c.c

```diff
@@ -30,12 +30,15 @@
 // Per #2 in the copyright notice above:
 // 
 // The definition of 'NONMINMAX' below was altered from the original.
+// cpuid.h and x86intrin.h are only included if we're on x86 or x64, fixing arm builds.
 
 //#define NOMINMAX
 
 #ifdef CRC32C_GCC
+#if defined(__x86_64__) || defined(__i386__)
 #include <cpuid.h>
 #include <x86intrin.h>
+#endif
```

Comment:
Maybe affect to tensorboard feature?

Reply:
I have no idea about how crc32c.c works so @NiklasGustafsson Can you look at this?

Reply:
HW support was removed from crc32c in [#2877164](https://github.com/dotnet/TorchSharp/commit/28771646190227c006c6df4494f82e929e791926), so this shouldn't affect anything. cpuid/x86intrin are only needed for the hardware crc function, which is no longer being used.

Reply:
Then, how about just remove that include lines?